src/compiler/crystal/macros.cr

```diff
@@ -87,6 +87,10 @@ module Crystal::Macros
   def raise(message) : NoReturn
   end
 
+  # Returns `true` if the given *filename* exists, `false` otherwise.
+  def file_exists?(filename) : BoolLiteral
```

Comment:
```suggestion
  def file_exists?(path) : BoolLiteral
```

Maybe `path` would be a better parameter name?

Reply:
I'd agree, although it's the same as in `read_file` macro, so 🤷‍♂️